OmniKit/PumpManager/PodComms.swift

```diff
@@ -25,6 +31,8 @@ class PodComms: CustomDebugStringConvertible {
 
     public let log = OSLog(category: "PodComms")
 
+    private var startingPacketNumber = 0
```

Comment:
The actual value for this variable is not that important, it has be able to change (not always start at the same packet #) to get the pod out of certain situations where the pod would otherwise be left in a permanent ack state. It does not need to be persistent.